app/src/components/searchWidget/Autocomplete.tsx

```diff
@@ -76,16 +76,16 @@ class Autocomplete extends React.Component<AutoCompleteProps, any> {
     }
 
     onSelectChanged(value: any, action: ActionMeta) {
-        this.props.onChange(this.props.defaultValue);
+        this.props.onChange(value);
     }
 
     render() {
         return (
             <AsyncSelect
                 inputId={this.props.id}
                 isClearable
-                defaultInputValue={this.props.defaultValue}
```

Comment:
Can't we set default value as 'MEL', so later on we'll just pass airport code?

Reply:
you mean the string 'MEL' over here itself?

Reply:
yeah, won't it work?

Reply:
Yes it will, that's how it was, just with the full name of the airport though, which I had removed and put in the .env file

Reply:
Righht, let me rewrite all 'any' for you, so you follow that pattern later on

Reply:
Sure, that would be super helpful. I mean, I'm able to fix the types for most of the stuff, but few variables here and there are new and I'm not able to find any fixes for them.